Core/Pixel.swift

```diff
@@ -180,19 +121,7 @@ public enum PixelName: String {
     case feedbackNegativePerformanceVideo = "mfbs_negative_performance_video"
     case feedbackNegativePerformanceOther = "mfbs_negative_performance_other"
     
-    case notificationOptIn = "m_ne"
-    case notificationOptOut = "m_nd"
-    
-    case brokenSiteReported = "m_bsr"
```

Comment:
We want to keep broken site reports, but this pixel doesn't appear to be used, instead using `brokenSiteReport` below.